File: Midnight Corn flakes 3_10_2025/my_discount.py
def my_discount(price, discount):
	new_price = price - (price*(discount/100))
	return new_price


commodity_price = int(input("Enter price: "))
seller_discount = int(input("Enter discount: "))

discounted_price = my_discount(commodity_price, seller_discount)
print("You new price is: ", discounted_price)

File: Midnight Corn flakes 3_10_2025/MyDiscount.java
import java.util.Scanner;

public class MyDiscount{
	public static void main(String[] args){
		Scanner input = new Scanner(System.in);
		double number1 = input.nextDouble();
		double number2 = input.nextDouble();
		double result = MyDsicount(number1, number2);
		System.out.printf("Your discounted price is: %d", result);
	}
	public static double MyDiscount(double input1, double input2){
		double discount = input1 - (input1 * (input2/100));
	}
}